lib/pbench/cli/agent/commands/stddev.py

```diff
@@ -0,0 +1,27 @@
+"""
+pbench-avg-stddev
+
+
+input: a bunch of numbers on cmd line
+output: an average (mean), sample standard deviation, sample standard deviation
+percent, and which input was closest to the average.
+
+nargs = -1, accepts an unlimited number of arguments
+"""
+
+import click
+
+import statistics
+
+
+@click.command()
+@click.argument("values", nargs=-1, type=click.FLOAT)
+def main(values):
+    values = list(values)
+    avg = statistics.mean(values)
+    stddev = statistics.stdev(values)
+    stddevpct = 100 * stddev / avg
+
+    closest_index = min(range(len(values)), key=lambda i: abs(values[i] - avg))
```

Comment:
Is this zero based, or one based indexing?  The original code appears to be one based.